api/app/utils/cffdrs.py

```diff
@@ -541,15 +542,20 @@ def crown_fraction_burned(fuel_type: str, fmc: float, sfc: float, ros: float, cb
     if cbh is None or fmc is None:
         message = PARAMS_ERROR_MESSAGE + \
             "_CFBcalc; fuel_type: {fuel_type}, cbh: {cbh}, fmc: {fmc}".format(
-                fuel_type=fuel_type, cbh=cbh, fmc=fmc)
+                fuel_type=fuel_type.value, cbh=cbh, fmc=fmc)
         raise CFFDRSException(message)
-    result = CFFDRS.instance().cffdrs._CFBcalc(FUELTYPE=fuel_type, FMC=fmc, SFC=sfc,
+    # FireBAT uses FuelTypeEnum, HFI Calc uses string to represent fuel type
```

Comment:
Can we just require callers to provide a `FuelTypeEnum` only to simplify?

Reply:
Yup, good call.